msal/src/cache/java/com/microsoft/identity/client/BaseTokenCacheItem.java

```diff
@@ -31,13 +31,13 @@
 abstract class BaseTokenCacheItem {
 
     @SerializedName("client_id")
-    final String mClientId;
+    String mClientId;
```

Comment:
i'm kind of against removing final. Final guarantees that those variables are only initialized in the constructor.